help/destinations/catalog/advertising/tradedesk-emails-update

```diff
@@ -0,0 +1,138 @@
+---
+title: (Beta) The Trade Desk - CRM connection
+description: Activate profiles to your Trade Desk account for audience targeting and suppression based on CRM data.
+exl-id: e09eaede-5525-4a51-a0e6-00ed5fdc662b
+---
+# (Beta) The [!DNL Trade Desk] - CRM connection
+
+>[!IMPORTANT]
+>
+>[!DNL The Trade Desk - CRM] destination in Platform is currently in beta. The documentation and functionality are subject to change. 
+>
+>With the release of EUID (European Unified ID), you are now seeing two [!DNL The Trade Desk - CRM] destinations in the Destinations catalog.
+>* if you source data in the EU, please use [!DNL The Trade Desk - CRM (EU)] destination. 
```

Comment:
Minor, let's start sentences after bullet points with capital letters, and add an article before [!DNL The Trade Desk - CRM (EU)]: 

* If you source data in the EU, please use the [!DNL The Trade Desk - CRM (EU)] destination.

Reply:
Thank you @vgiurgiu ! I have capitalized the start of the bullet points, but can you clarify what you mean by adding an article?